docs/12-design.md

```diff
@@ -55,23 +55,22 @@ sometimes be used to avoid reading a lot of the data).
 ## Tables
 
 All records in a table conform to a schema. The records in a table are stored in multiple files, with each file
-belonging to a partition. These files are all in an S3 bucket that is exclusively used by this table.
+belonging to a partition. These files are all stored in an S3 bucket deployed at an instance level, referred to as 
+the data bucket.
 
 Each table has a state store associated to it. This stores metadata about the table, namely the files that are in
 the table and how the records in the table are partitioned.
 
-Tables are deployed by the CDK table stack. This stack creates the infrastructure for each table. Each table
-requires a bucket where its data will be stored, and a state store. When a table is first created, its state store
-must be initialised.
+Table infrastructure is deployed at an instance level by the CDK table stacks. These stacks creates the infrastructure 
+for all tables. All tables require an S3 bucket to store data and a state store to store metadata about files in each table. 
+When a table is first created, its state store must be initialised. To achieve this, both state store implementations 
+support storing metadata for files in all tables. 
 
-To achieve this, the table stack obtains a list of the table properties files from the instance properties. For
-each table properties file, it creates the data bucket for that table and creates the state store. The creation
-of the state store is delegated to either the DynamoDB state store stack or the S3 state store stack, as appropriate.
-A custom CDK resource is then used to call `sleeper.cdk.custom.SleeperTableLambda`. This lambda initialises the
-state store and updates the table properties file which is stored in the instance's config bucket.
+The directory in the data bucket which contains a table's data is constructed from the table name:
+`<data-bucket>/<tablename>/<file>`, e.g. `sleeper-mySleeperInstance-data/table1/file1.parquet`.
```

Comment:
The directory actually uses the table ID rather than the name. Maybe we should avoid stating this explicitly, since we wouldn't want to use the table ID directly. It is mentioned earlier that each table has its own folder.